kolibri/plugins/learn/assets/src/views/HybridLearningContentCardListView.vue

```diff
@@ -1,86 +1,60 @@
-          />
-        </p>
-        <div v-if="displayCategoryAndLevelMetadata && !isMobile" class="metadata-info">
-          <p> {{ coreString(displayCategoryAndLevelMetadata) }}</p>
-        </div>
-        <img
-          v-if="!isMobile"
-          :src="content.channel_thumbnail"
-          :alt="learnString('logo', { channelTitle: content.channel_title })"
-          class="channel-logo"
-        >
-        <KButton
-          v-if="!isMobile && content.copies && content.copies.length"
-          appearance="basic-link"
-          class="copies"
-          :style="{ color: $themeTokens.text }"
-          :text="coreString('copies', { num: content.copies.length })"
-          @click.prevent="$emit('openCopiesModal', content.copies)"
-        />
-      </span>
-    </router-link>
-    <KFixedGrid :numCols="10" class="footer">
-      <KFixedGridItem span="6" class="footer-elements">
-        <p
-          v-if="isBookmarksPage"
-          class="metadata-info-footer"
-          :style="{ color: $themePalette.grey.v_700 }"
-        >
-          {{ bookmarkCreated }}
-        </p>
-        <ProgressBar :contentNode="content" />
-      </KFixedGridItem>
-      <KFixedGridItem span="3" alignment="right" class="footer-elements footer-icons">
+          <p
+            v-if="isBookmarksPage"
+            class="created-info"
+            :style="{ color: $themePalette.grey.v_700 }"
+          >
+            {{ bookmarkCreated }}
+          </p>
+          <ProgressBar :contentNode="content" />
+        </KFixedGridItem>
+
+        <KFixedGridItem :span="isMobile ? 4 : 3" class="text-area">
+          <span :style="{ color: $themeTokens.text }">
+            <div class="metadata-info" :style="{ color: $themePalette.grey.v_700 }">
+              <LearningActivityLabel :contentNode="content" labelAfter condensed />
+            </div>
+            <h3>
+              <TextTruncatorCss :text="content.title" :maxLines="1" />
+            </h3>
+            <p v-if="content.description" style="font-size: 14px;">
+              <TextTruncatorCss :text="content.description" :maxLines="4" />
+            </p>
+            <div v-if="!isMobile" class="bottom-items">
+              <p v-if="categoryAndLevelString">{{ categoryAndLevelString }}</p>
+              <img
+                :src="content.channel_thumbnail"
+                :alt="learnString('logo', { channelTitle: content.channel_title })"
+                class="channel-logo"
+              >
+              <KButton
+                v-if="content.copies && content.copies.length"
+                appearance="basic-link"
+                class="copies"
+                :style="{ color: $themeTokens.text }"
+                :text="coreString('copies', { num: content.copies.length })"
+                @click.prevent="$emit('openCopiesModal', content.copies)"
+              />
+            </div>
+          </span>
+        </KFixedGridItem>
+      </KFixedGrid>
+
+      <div class="footer">
```

Comment:
because the footer needs to be pinned to the bottom of the card, it's still implemented using an absolutely-positioned `div`, not a `KGridItem`